.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: Node.js Package
```

Comment:
@aleschneck Could this have a better name? Like "Publish to npm" or something

Reply:
it certainly can! cheers!